packages/apollo-angular/src/apollo.ts

```diff
@@ -68,27 +96,29 @@ export class ApolloBase<TCacheShape = any> {
     TFragmentData = unknown,
     TVariables extends OperationVariables = EmptyObject,
   >(
-    options: WatchFragmentOptions<TFragmentData, TVariables>,
-    extra?: ExtraSubscriptionOptions,
```

Comment:
I went ahead and combined these extra options with the `options` object so that its a single argument. Happy to revert if you prefer the `extra` options as a separate argument (didn't feel like it needed to be separate to me though).